### PR TITLE
hotfix editor csrf

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

File: src/components/pages/add-revision.tsx [hunk 1]
```diff
@@ -16,6 +17,23 @@ export function AddRevision({
...
+  useEffect(() => {
+    if (window.location.hostname === 'localhost') {
+      setCookieReady(true)
+    } else {
+      fetch('/auth/password/change')
+        .then((res) => res.text())
+        .then(() => {
+          setCookieReady(true)
+        })
+        .catch(() => {})
```
Comment (inyono) [inline src/components/pages/add-revision.tsx:31]: Possible optimization: check if the cookie already exists.

**Review by inyono — COMMENTED** [review]
LGTM

File: src/components/pages/add-revision.tsx (hunk 1 above)
Comment (elbotho) [reply src/components/pages/add-revision.tsx:31]: sure it can not be outdated or smth?

Comment (elbotho) [reply src/components/pages/add-revision.tsx:31]: will add optimization later via staging.

**Pull request merged**